### PR TITLE
feat(Moved slider component from core to plugin-react-native-paper)

### PR DESCRIPTION
**Pull request by taha437**
## Description

I removed slider component from BlueBaseJS/core and moved it to BlueBaseJS/plugin-react-native-paper.

## Motivation and Context

Support for slider component is going to be removed from react-native v0.60.0. So we decided to remove it from BlueBaseJS/core and move it to our BlueBaseJS/plugin-react-native-paper so it doesn't affect our functionality.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -132,7 +139,7 @@
     "npm-run-all": "^4.1.3",
     "opn-cli": "^4.0.0",
     "prettier": "^1.16.4",
-    "react": "^16.8.6",
+    "react": "^16",
```
Comment (artalat) [inline package.json]: Revert this

File: package.json [hunk 2]
```diff
@@ -55,6 +51,16 @@
     "web:start": "bluebase web:start",
     "expo:start": "bluebase expo:start"
   },
+  "name": "@bluebase/plugin-react-native-paper",
```
Comment (artalat) [inline package.json]: Move these lines back at the top

File: package.json [hunk 3]
```diff
@@ -1,17 +1,13 @@
...
-  "main": "dist/index.js",
-  "typings": "dist/index.d.ts",
-  "module": "dist/index.js",
-  "repository": "https://github.com/BlueBaseJS/plugin-react-native-paper",
-  "license": "MIT",
-  "keywords": [],
   "config": {
     "react-native-storybook-loader": {
       "searchDir": [
+        "src",
```
Comment (artalat) [inline package.json]: Remove duplication

File: bluebase/storybook-native/storybook/CenterView/style.js [hunk 4]
```diff
@@ -0,0 +1,8 @@
+export default {
```
Comment (artalat) [inline bluebase/storybook-native/storybook/CenterView/style.js]: Delete this file. We don't need it

File: bluebase/storybook-native/storybook/CenterView/index.js [hunk 5]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
```
Comment (artalat) [inline bluebase/storybook-native/storybook/CenterView/index.js]: Delete this file. We don't need it

File: bluebase/storybook-native/sample/index.ts [hunk 6]
```diff
@@ -0,0 +1,12 @@
+import DummyComponent from './DummyComponent';
```
Comment (artalat) [inline bluebase/storybook-native/sample/index.ts]: Delete this file. We don't need it

File: bluebase/storybook-native/sample/DummyComponent.stories.tsx [hunk 7]
```diff
@@ -0,0 +1,12 @@
+declare var module: any;
```
Comment (artalat) [inline bluebase/storybook-native/sample/DummyComponent.stories.tsx]: Delete this file. We don't need it.

File: bluebase/storybook-native/sample/README.md [hunk 8]
```diff
@@ -0,0 +1,8 @@
+# Sample Code
```
Comment (artalat) [inline bluebase/storybook-native/sample/README.md]: Delete this file. We don't need it.

File: bluebase/storybook-native/sample/DummyComponent.tsx [hunk 9]
```diff
@@ -0,0 +1,6 @@
+import { Text } from '@bluebase/core';
+import React from 'react';
```
Comment (artalat) [inline bluebase/storybook-native/sample/DummyComponent.tsx]: Delete this file. We don't need it.

File: bluebase/storybook-native/bluebase.ts [hunk 10]
```diff
@@ -1,17 +1,22 @@
...
 /**
  * Add your platform specific configs here.
  * We keep all the universal (cross platform) configs in
  * the common folder, and extend them here.
  */
 const bootOptions: Partial<BootOptions> = {
 
-	plugins: [ReactnativePaperPlugin,MaterialCommunityIcons],
+	plugins: [
+		// TODO: Only for evaluation, remove this
```
Comment (artalat) [inline bluebase/storybook-native/bluebase.ts]: Revert changes in this file

Comment (taha437) [reply bluebase/storybook-native/bluebase.ts]: Done All

File: tsconfig.json [hunk 11]
```diff
@@ -9,7 +9,7 @@
 			"*": ["typings/*"]
 		}
 	},
-	"include": ["src/**/*.ts"],
+	"include": ["src/**/*.ts", "src/components/Slider/Slider.tsx", "src/components/Slider/index.tsx"],
```
Comment (artalat) [inline tsconfig.json]: why was this done?

Comment (taha437) [reply tsconfig.json]: Sorry, was just checking something with the file extensions. Had no idea tsconfig.json added these lines by itself. Fixed it and pushed

Comment (artalat) [reply tsconfig.json]: always review each file before committing.

File: package.json (hunk 1 above)
Comment (taha437) [reply package.json]: Done.

File: package.json (hunk 2 above)
Comment (taha437) [reply package.json]: Done.

File: package.json (hunk 3 above)
Comment (taha437) [reply package.json]: Done.

File: bluebase/storybook-native/storybook/CenterView/style.js (hunk 4 above)
Comment (taha437) [reply bluebase/storybook-native/storybook/CenterView/style.js]: Done.

File: bluebase/storybook-native/storybook/CenterView/index.js (hunk 5 above)
Comment (taha437) [reply bluebase/storybook-native/storybook/CenterView/index.js]: Done

File: bluebase/storybook-native/sample/index.ts (hunk 6 above)
Comment (taha437) [reply bluebase/storybook-native/sample/index.ts]: Done.

File: bluebase/storybook-native/sample/DummyComponent.stories.tsx (hunk 7 above)
Comment (taha437) [reply bluebase/storybook-native/sample/DummyComponent.stories.tsx]: Done.

File: bluebase/storybook-native/sample/README.md (hunk 8 above)
Comment (taha437) [reply bluebase/storybook-native/sample/README.md]: Done.

File: bluebase/storybook-native/sample/DummyComponent.tsx (hunk 9 above)
Comment (taha437) [reply bluebase/storybook-native/sample/DummyComponent.tsx]: Done.

File: tsconfig.json (hunk 11 above)
Comment (taha437) [reply tsconfig.json]: Ok sir.

**Pull request merged**